docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,498 @@
+
+    1.  Select the **Template** project type and click **Next**.
+
+        ![Selecting the project type](media/create_project_wizard_type.png)
+
+    2.  Select the profile (**Mobile**) and version from a drop-down list and click **Next**.
+
+        The version depends on the platform version you have installed and with which you are developing the addon.
+
+        ![Selecting the profile and version](media/create_project_wizard_version.png)
+
+    3.  Select the **Web Application** application type and click **Next**.
+
+        ![Selecting the application type](media/create_project_wizard_app_web.png)
+
+    4.  Select the **Basic Addon** template and click **Next**.
+
+        ![Selecting the template](media/create_project_wizard_template_ma.png)
+
+    5.  Define the project properties and click **Finish**.
+
+        You can enter the project name (3-50 characters) and the unique package ID. You can also select the location and working sets by clicking **More properties**.
+
+        ![Defining properties](media/create_project_wizard_properties_ma.png)
+
+        The Project Wizard sets up the project, creates the addon files using the default content from the template, and closes. For more information on the Project Wizard and the available templates, see [Creating Tizen Projects with Tizen Project Wizard](../../../tizen-studio/web-tools/project-wizard.md).
+
+You can see the created project in the **Project Explorer** view. The most important files and folders include:
+
+-   `shared/res`: Folder for addon files including the manifest file and JavaScript files.
+
+-   `config.xml`: Addon configuration file used by the platform to install and launch the addon.
+
+**Figure: Addon in the Project Explorer**
+
+![Addon in the Project Explorer](media/basic_addon_project_explorer_ma.png)
+
+> [!NOTE]
+> You can [view and modify the addon configuration](#configuration) in the addon configuration editor. In this example, no configuration changes are required.
+
+Your addon project is now ready for further actions. Next, build the addon.
+
+<a name="configuration"></a>
+### Manage addon configuration
+
+To view and modify the addon configuration:
+
+1.  In the **Project Explorer** view, double-click the `config.xml` file of the addon. Tizen Studio opens the file in the addon configuration editor.
+
+2.  In the configuration editor, view and modify the configuration details using the various tabs:
+
+    ![Configuring the addon](media/basic_addon_config_w.png)
+    > [!NOTE]
+    > The `config.xml` file must conform to both the XML file format and the W3C specification requirements. Editing the file in the **Source** tab is intended for advanced users only.  
+    > You can see an error in the captured screen above. However, it does not affect on this process because the contents field is not used for addons.
```

Comment:
@mijin85cho, it is correct that the directional language ('above') is not suggested.
Suggest rephrasing to:

> You can see an error in the preceding image of step 2. However, it does not affect this process because **Content** is not used for addons.

Please verify

Reference for preceding: Google Style Guide (https://developers.google.com/style/word-list?hl=bg)
above
Don't use. Instead, use preceding.

Reply:
Done